### PR TITLE
atabek (week1)

### PR DESCRIPTION
**Pull request by atabek-bakhramov**

### CODE REVIEW

File: week1/homework/exercises/2-left-pad/script.js [hunk 1]
```diff
@@ -8,6 +8,19 @@
...
\ No newline at end of file
+// YOUR CODE GOES HERE
+
+// This code snippet follows the requirements of the exercise 2:
+// const leftPad = require('left-pad');
+// numbers.forEach(element => {
+//   console.log(leftPad(element, 5, '_'));
+// });
+// But the left-pad module is deprecated.
+
```
Comment (gajduk) [inline week1/homework/exercises/2-left-pad/script.js:21]: ok maybe we need to update out exercise :)

Comment (gajduk) [reply week1/homework/exercises/2-left-pad/script.js:21]: nice of you to notice that

File: week1/homework/exercises/3-web-server/server.js [hunk 2]
```diff
@@ -2,13 +2,43 @@
...
+			'.js': 'text/javascript',
+			'.css': 'text/css',
+			'.json': 'application/json',
+			'.png': 'image/png',
+			'.jpg': 'image/jpg'
+		};
+
+		if (err) {
+			if (err.code === 'ENOENT') {
+				fs.readFile(path.join(__dirname, 'public', '404.html'), (err, content) => {
```
Comment (gajduk) [inline week1/homework/exercises/3-web-server/server.js:25]: Excellent work with the error handling and the content type mapping.
I am impressed. Here a few more things that you  can improve:
- in case of an error log the error to the console and send in the response only something went wrong, no detials
- make the 404 response into a function, nesting callbacks is so hard to follow. Also dont forget about errors when reading the 404 file
- no reason not to map html to content type

**Review by gajduk — APPROVED** [review]
Excellent work!

File: week1/homework/exercises/2-left-pad/script.js (hunk 1 above)
Comment (atabek-bakhramov) [reply week1/homework/exercises/2-left-pad/script.js:21]: thank you :) a new exercise is coming then? 😁

File: week1/homework/exercises/3-web-server/server.js (hunk 2 above)
Comment (atabek-bakhramov) [reply week1/homework/exercises/3-web-server/server.js:25]: Thank you very much for pointing these things out! Let me work on them and once I complete the exercise, with your permission I'll take some of your time on slack? 😁

**Comment by atabek-bakhramov** [comment]
> Excellent work!

Thank you very much for your feedback! Much appreciated!
